### PR TITLE
Enhancement: Updates Sonata Media Bundle

### LINKED ISSUE #95: Update Sonata Version
https://github.com/sonata-project/SonataMediaBundle/issues/2209

### PR DESCRIPTION
**Pull request by silasjoisten**
* Updates Sonata Media Bundle
* Moves MultiUploadAction to seperate controller.
* Overrides MediaAdminController::createAction()

closes #95

### CODE REVIEW

File: src/Controller/MultiUploadController.php [hunk 1]
```diff
@@ -2,66 +2,49 @@
...
         $form = $this->createMultiUploadForm($provider, $context);
         if (!$request->files->has('file')) {
-            return $this->render('@SonataMultiUpload/multi_upload.html.twig', [
+            return $this->responder->render('@SonataMultiUpload/multi_upload.html.twig', [
                 'action' => 'multi_upload',
                 'form' => $form->createView(),
                 'provider' => $provider,
-                'maxUploadFilesize' => $this->container->getParameter('sonata_multi_upload.max_upload_filesize'),
-                'redirectTo' => $this->container->getParameter('sonata_multi_upload.redirect_to'),
+                'maxUploadFilesize' => $this->maxUploadSize,
```
Comment (OskarStark) [inline src/Controller/MultiUploadController.php:46]:
```suggestion
                'maxUploadFilesize' => $this->maxUploadFilesize,
```
To match the key?
Just a proposal

File: src/Resources/config/controller.yaml [hunk 2]
```diff
@@ -1,4 +1,17 @@
 services:
     SilasJoisten\Sonata\MultiUploadBundle\Controller\MultiUploadController:
-        arguments: ['@sonata.media.manager.media']
+        arguments:
+            $mediaManager: '@sonata.media.manager.media'
```
Comment (OskarStark) [inline src/Resources/config/controller.yaml:4]: You should add a recipe for this bundle 👍🏻

Comment (silasjoisten) [reply src/Resources/config/controller.yaml:4]: There is already a receipe

File: src/Controller/MultiUploadController.php (hunk 1 above)
Comment (silasjoisten) [reply src/Controller/MultiUploadController.php:46]: Well better to fix the key, the config is also called like that.

File: .github/workflows/ci-php.yaml [hunk 3]
```diff
@@ -139,4 +139,4 @@ jobs:
 
             -
                 name: "Run unit tests with phpunit/phpunit"
-                run: "vendor/bin/phpunit --testsuite=unit --coverage-clover=.build/phpunit/coverage-unit.xml"
+                run: "SYMFONY_DEPRECATIONS_HELPER=weak vendor/bin/phpunit --testsuite=tests --coverage-clover=.build/phpunit/coverage-unit.xml"
```
Comment (OskarStark) [inline .github/workflows/ci-php.yaml:142]: env could be defined in line 16

File: .gitignore [hunk 4]
```diff
@@ -7,4 +7,9 @@ npm-debug.log
 yarn-error.log
 
 /phpunit.xml
+.phpunit.result.cache
+
 composer.lock
+
+.php-version
```
Comment (OskarStark) [inline .gitignore:14]: should be added do .gitattributes as well

File: composer.json [hunk 5]
```diff
@@ -11,8 +11,11 @@
     ],
     "require": {
         "php": "^8.0",
-        "sonata-project/admin-bundle": "^3.35",
-        "sonata-project/media-bundle": "^3.18"
+        "oskarstark/symfony-http-responder": "^0.3.0",
```
Comment (OskarStark) [inline composer.json:14]: ❤️

**Pull request merged**